res/controllers/DJ TechTools MIDI Fighter Twister.midi.xml

```diff
@@ -2,286 +2,439 @@
 <MixxxControllerPreset mixxxVersion="" schemaVersion="1">
   <info>
     <name>DJ TechTools MIDI Fighter Twister</name>
-    <author>tock203</author>
-    <description>for 2 deck use</description>
-    <manual>dj_techtools_midi_figher_twister</manual>
+    <author>tock203, Sam Whited</author>
+    <description>Control 4 decks with the MIDI Fighter Twister.</description>
+    <manual>dj_techtools_midi_fighter_twister</manual>
+    <forums>https://mixxx.discourse.group/t/mapping-the-midifighter-twister/16253</forums>
+    <devices>
+      <product protocol="midi" vendor_id="0x2580" product_id="0x0007"/>
+    </devices>
   </info>
+  <settings>
+    <option variable="vuMeter" type="boolean" label="Use gain knob as vu meter" default="false">
+      <description>
+        Show instantaneous volume on the 11 segment LEDs around the gain knob instead of the position of the knob.
+      </description>
+    </option>
+    <option variable="beatColor" type="enum" label="Beat Indicator Color">
+      <description>
+        Use the tempo knob LED as a beat indicator when set.
+      </description>
+      <value label="Off" default="true">off</value>
+      <value label="Firmware Default">127</value>
+      <value label="Blue">1</value>
+      <value label="Celeste">32</value>
+      <value label="Teal">40</value>
+      <value label="Green">50</value>
+      <value label="Lime">60</value>
+      <value label="Yellow">66</value>
+      <value label="Orange">74</value>
+      <value label="Red">85</value>
+      <value label="Fuscia">100</value>
+      <value label="Purple">108</value>
```

Comment:
Just a quick note that [`value` now support the `color` attribute](https://github.com/mixxxdj/mixxx/wiki/MIDI-Scripting#additional-attributes-and-elements-for-the-enum-type). Just a nit but it can help enriching the UX 
![image](https://github.com/user-attachments/assets/e8d74a39-6c8e-43d8-a4bf-a409e1ee0fdb)

Reply:
Oh right; is that 2.6 only, and should I be switching these branches over to use 2.6 as a base since that's in beta and presumably going to be released in the near future?

Reply:
ooh, wait, this isn't the color type that was added in 2.6 (I think), this is an attribute for the enum type, got it! Thanks, I'll go add those!

Reply:
Added; thanks again for the tip! This is only supported in 2.6, but it's an attribute that will be ignored otherwise so might as well go ahead and put this in here even if it gets merged into 2.5.2 eventually.